src/commands/ml-models-get.ts

```diff
@@ -1,10 +1,54 @@
+import * as clc from "cli-color";
 import { Command } from "../command";
+import * as getProjectId from "../getProjectId";
 import { FirebaseModel } from "../ml/models";
+import { requirePermissions } from "../requirePermissions";
+import { ensureFirebaseMlApiEnabled, isValidModelId, getTableForModel } from "../ml/mlHelper";
+import * as mlApi from "../ml/mlApi";
+import { FirebaseError } from "../error";
+import * as logger from "../logger";
 
 export default new Command("ml:models:get <modelId>")
-  .description("gets the Firebase ML Model with the given modelId")
+  .description("get the Firebase ML Model with the given modelId")
+  .help("TODO")
+  .before(requirePermissions, ["firebaseml.models.get"])
+  .before(ensureFirebaseMlApiEnabled)
   .action(
-    async (modelId: string): Promise<FirebaseModel> => {
-      throw new Error("Not Implemented");
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    async (modelId: string, options: any): Promise<FirebaseModel> => {
+      if (!isValidModelId(modelId)) {
+        throw new FirebaseError("Must specify a valid model ID");
+      }
+
+      const projectId = getProjectId(options);
+      let model: FirebaseModel;
+      try {
+        model = await mlApi.getModel(projectId, modelId);
+      } catch (err) {
+        if (err.status === 404) {
+          return Promise.reject(
+            new FirebaseError(`No model ${modelId} in project ${projectId}.`, options)
+          );
```

Comment:
I believe throwing the error is sufficient - I don't think it has to be wrapped in `Promise.reject`

Reply:
done